gcp/monitoring_metric.go

```diff
@@ -56,6 +56,11 @@ func commonMonitoringMetricColumns() []*plugin.Column {
 			Description: "The sum of the metric values for the data point.",
 			Type:        proto.ColumnType_DOUBLE,
 		},
+		{
+			Name:        "time_stamp",
```

Comment:
@Subhajit97 I think `timestamp` is normally one word, so I'd expect the column name to be `timestamp`, and the description to be `The timestamp used ...`

Does GCP have it as separate words?